retrace-server.spec

```diff
@@ -171,7 +171,7 @@ exit 0
 %config(noreplace) %{_sysconfdir}/%{name}/hooks/start.conf
 %config(noreplace) %{_sysconfdir}/%{name}/hooks/success.conf
 %config(noreplace) %{_sysconfdir}/%{name}/hooks/task.conf
-%config(noreplace) %{_sysconfdir}/logrotate.d/%{name}
```

Comment:
It should be either

%{_sysconfdir}/logrotate.d/

or 

Requires: logrotate